templates/spec-s2i.tpl

```diff
@@ -19,6 +19,11 @@ params:
     type: string
     description: |
       Fully qualified container image name to be built by s2i.
+  - name: IMAGE_SCRIPTS_URL
+    type: string                                   #default: image:///usr/libexec/s2i         
+    description: |
+      URL containing the default assemble and run scripts for the builder image.
```

Comment:
Instead, let's copy and paste what `s2i` help message shows:

```
$ s2i build --help |grep image-scripts-url
      --image-scripts-url string         Specify a URL containing the default assemble and run scripts for the builder image (default "image:///usr/libexec/s2i"
```

Thus:

```yaml
- name: IMAGE_SCRIPTS_URL
  type: string
  default: image:///usr/libexec/s2i         
  description: |
    Specify a URL containing the default assemble and run scripts for the
    builder image
```

Be careful with spaces and extra comments at the end of lines, please. 🙏